### ISSUE TITLE
Better node collapsing.

### ISSUE THREAD

**Issue by ratherrandomaccname9000** [RodZill4/material-maker#529]
So it would be nice to have that collapse square in the right corner of each node to additionally reduce its opacity and maybe change color so that it clearly makes visual difference between nodes I want to see and use versus those that are there just to never be touched again.

**Comment by rubberduck9999** [comment]
yeah, I would also like beeing able to change / overwrite colors, or for example to set a custom name for nodes I want to (though for many I can do it in the way of making them editable, but for some it isn't possible, like switch or image).

I would also consider renaming this issue to something like "better node customisation" or "make nodes customizable (collapsing, custom colors and text, etc)", instead of opening a new issue here.
